resources/test/dumps/Sandpile.js

```diff
@@ -245,9 +245,11 @@ function dropPatch() {
     return ListPrims.oneOf(world.patches());
   }
   if ((Prims.equality(world.observer.getGlobal('drop-location'), "mouse-click") && MousePrims.isDown())) {
-    Prims.every(0.3, function() {
+    var dropPatch_0;
+    if (Prims.everyTimerElapsed(0.3, dropPatch_0)) {
       return world.getPatchAt(MousePrims.getX(), MousePrims.getY());
-    }, 'dropPatch-0');
+      dropPatch_0 = new Prims.everyTimer();
+    }
```

Comment:
It does pass the tests which I wrote for every, but they apparently don't cover all possible behavior. This implementation of every does work in loops (at least loops that don't have early exit), which is (unsurprisingly) what the tests of every are based around.

In NetLogo-Headless, `every` is scoped by command, agent, and job. It is scoped by command here (although that scoping has some problems). I can fix the problems and add in agent-scoping. What is the tortoise analog of a job that I should use for scoping every?